irohad/ametsuchi/impl/postgres_ordering_service_persistent_state.cpp

```diff
@@ -69,20 +60,34 @@ namespace iroha {
           log_(logger::log("PostgresOrderingServicePersistentState")),
           execute_{ametsuchi::makeExecuteResult(*postgres_transaction_)} {}
 
-    void PostgresOrderingServicePersistentState::initStorage() {
+    bool PostgresOrderingServicePersistentState::initStorage() {
+      bool result = true;
       log_->info("Init storage");
-      postgres_transaction_->exec(
-          "CREATE TABLE IF NOT EXISTS ordering_service_state (\n"
-          "    proposal_height bigserial\n"
-          ");"
-          "INSERT INTO ordering_service_state\n"
-          "VALUES (2); -- expected height (1 is genesis)");
+      try {
+        postgres_transaction_->exec(
```

Comment:
`postgres_wsv_common.hpp` contains functional style wrapper for transactions. Better to rework this.